### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by NhanAZ**

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -3,9 +3,6 @@ name: HideCommands
 version: 2.0.1
 main: Himbeer\HideCommands\Main
 api: 4.0.0
-mcpe-protocol:
```
Comment (HimbeersaftLP) [inline plugin.yml:6]: The plugin will not get accepted on Poggit when packets are used directly and the protocol isn't specified

Comment (HimbeersaftLP) [inline plugin.yml:6]: The plugin will not get accepted on Poggit when packets are used directly and the protocol isn't specified

File: src/Himbeer/HideCommands/Main.php [hunk 2]
```diff
@@ -22,19 +22,13 @@ class Main extends PluginBase implements Listener {
...
-				$this->mode = self::MODE_BLACKLIST;
-				break;
-			default:
-				$this->getLogger()->error('Invalid mode selected, must be either "blacklist" or "whitelist"! Disabling...');
-				return;
-		}
+		match ($this->getConfig()->get("mode", "whitelist")) {
+			"whitelist" => $this->mode = self::MODE_WHITELIST,
+			"blacklist" => $this->mode = self::MODE_BLACKLIST,
+			default => $this->getLogger()->error('Invalid mode selected, must be either "blacklist" or "whitelist"! Disabling...')
```
Comment (HimbeersaftLP) [inline src/Himbeer/HideCommands/Main.php:28]: The "return" from the previous code is missing here, so the listener would still be registered

File: src/Himbeer/HideCommands/Main.php [hunk 3]
```diff
@@ -22,19 +22,13 @@ class Main extends PluginBase implements Listener {
...
+		};
 
-		foreach ($this->getConfig()->get("commands") as $command) {
+		foreach ($this->getConfig()->get("commands", ["op", "gamemode", "version"]) as $command) {
```
Comment (HimbeersaftLP) [inline src/Himbeer/HideCommands/Main.php:31]: Undocumented defaults are not good

**Comment by HimbeersaftLP** [comment]
I appreciate your pull request, however for the reasons mentioned above, I cannot merge it.

**Comment by NhanAZ** [comment]
Thanks for the review!

**Pull request closed without merging**